target/linux/x86/generic/config-6.6

```diff
@@ -31,7 +32,7 @@ CONFIG_ACPI_TAD=y
 CONFIG_ACPI_THERMAL=y
 # CONFIG_ACPI_TOSHIBA is not set
 CONFIG_ACPI_VIDEO=y
-# CONFIG_ACPI_WMI is not set
+CONFIG_ACPI_WMI=y
```

Comment:
Seeing this a bunch.  This doesn't seem appropriate for a _refresh_.  I can believe it is necessary, just doesn't seem right for this commit.

Reply:
fixed